server/src/main/scala/org/http4s/server/ServerApp.scala

```diff
@@ -34,8 +34,8 @@ trait ServerApp extends ProcessApp {
   def shutdown(server: Server): Task[Unit] =
     server.shutdown
 
-  final def main(args: List[String]): Process[Task, Nothing] =
+  final def process(args: List[String]): Process[Task, Unit] =
     Process.bracket(server(args))(s => Process.eval_(s.shutdown)) { s =>
-      Process.eval_(Task.async[Nothing](_ => ()))
+      Process.eval_(Task.async[Unit](_ => ()))
```

Comment:
It never emits.

I'm trying to remember why `ProcessApp` returns Unit instead of Nothing.  I feel like this has flipped back and forth.